lib/appsignal/config.ex

```diff
@@ -86,6 +89,41 @@ defmodule Appsignal.Config do
     map
   end
 
+  defp skip_session_data_backwards_compatibility(config, nil) do
+    if Map.has_key?(config, :send_session_data) do
+      Map.put(config, :skip_session_data, !config[:send_session_data])
+    else
+      config
+      |> Map.merge(%{send_session_data: true, skip_session_data: false})
```

Comment:
Should these be added to sources as well? When `:skip_session_data` is added based on `:send_session_data`, it should probably be "system" (or "override", per [integration-guide#128](https://github.com/appsignal/integration-guide/pull/128)), but on the else branch, when both are unset, I think it should be "default".

On the integration guide document that describes the config options and their sources, it doesn't explicitly say that config options should always come from a source. This might be worth adding to it?

Reply:
Yes, but also no. Let's add it to the ["override" source](https://github.com/appsignal/integration-guide/pull/128), rather than default, as it may be overridden by this logic. The "default" source should be static.

> On the integration guide document that describes the config options and their sources, it doesn't explicitly say that config options should always come from a source. This might be worth adding to it?

I'll add something